src/migrations/tasks/09.add-last-updated-data.ts

```diff
@@ -1,45 +1,41 @@
-import { baseConfig } from "../../models/Config";
-import { D2Api } from "../../types/d2-api";
-import { addAttributeValueToObj } from "../../models/Attributes";
+import { D2Api, Id } from "../../types/d2-api";
 import { Debug, Migration } from "../types";
 import { post } from "./common";
+import { getUid } from "../../utils/dhis2";
 
 class AddLastUpdatedDataMigration {
     constructor(private api: D2Api, private debug: Debug) {}
 
-    async execute() {
-        await this.updateOrgUnits();
+    private async post<Payload>(payload: Payload) {
+        post(this.api, this.debug, payload);
```

Comment:
There should be a `await` here, otherwise it returns and the parent `await` has no effect. So if there is an error in the POST, it won't be cached and the migration will marked as successful. `return post...` should also work.